go.work

```diff
@@ -109,6 +109,11 @@ use (
 	pkg/networkdevice/profile
 	pkg/networkpath/payload
 	pkg/obfuscate
+	pkg/opentelemetry-mapping-go/inframetadata
+	pkg/opentelemetry-mapping-go/inframetadata/gohai/internal/gohaitest
+	pkg/opentelemetry-mapping-go/otlp/attributes
+	pkg/opentelemetry-mapping-go/otlp/logs
+	pkg/opentelemetry-mapping-go/otlp/metrics
```

Comment:
Why do those five need to be different modules ? Why not just `pkg/opentelemetry-mapping-go` ?

Reply:
I kept the same modules as before.